scripts/translate_schemas.py

```diff
@@ -40,7 +40,7 @@ def translate_schemas(runner_directory):
             schema_path = f"{runner_directory}/schemas/en/{schema_file}"
 
             handle_translate_schema(
-                schema_path, f"{language_dir}/{translation_file}", f"{output_dir}"
+                schema_path, f"{language_dir}/{translation_file}", output_dir, language
```

Comment:
I think this needs to be updated now that `language` is no longer required.

Reply:
👍 was waiting for other PR to be approved first. Pipenv doesn't seem to install the latest commit of the branch, hence why build is also failing.

Reply:
Up to date now